deploy/olm-catalog/mig-operator/latest/migstorage.crd.yaml

```diff
@@ -48,17 +48,24 @@ spec:
                   type: string
                 azureStorageAccount:
                   type: string
+                azureStorageContainer:
+                  type: string
                 credsSecretRef:
                   type: object
                 gcpBucket:
                   type: string
+                s3CustomCABundle:
+                  format: byte
+                  type: string
               type: object
             backupStorageProvider:
               type: string
             volumeSnapshotConfig:
               properties:
                 awsRegion:
                   type: string
+                awsSnapshotCreationTimeout:
+                  type: string
```

Comment:
This field will change shortly, but since this reflects what's currently on master, it's probably fine to leave it.